doc/manpages/man1/nbp.1.md

```diff
@@ -50,6 +53,15 @@ Find all devices of type *LaserWriter* in the local zone.
                   Hamtramck:LaserWriter        7942.2:134
              Iron Mountain :LaserWriter        7942.128:250
     example%
+    
+Find all devices of type *netatalk* in the local zone, providing script-friendly
+output.
+
+	example% nbplkup -s :netatalk
+	5.42:4 netatalk-build:netatalk
+	4.162:4 prometheus:netatalk
+	8.31:4 Tiryns:netatalk
+	example%
```

Comment:
The commonmark spec expects four space indentation for code blocks, rather than tabs.

https://spec.commonmark.org/0.31.2/#indented-code-blocks

GitHub seems to still parse this tab indentation correctly, but there's no guarantee that all cmark parsers will.